publication/ver11/societal_impact.md

```diff
@@ -0,0 +1,216 @@
+
+The WoT standards primarily describe metadata and how to access it.  To mitigate
+failures, such as the inability to access metadata if a WoT Discovery service
+is unavailable, such systems should use WoT Discovery and metadata (WoT Thing
+Descriptions) for initial connections and if updated metadata is not available,
+use the last known state of this metadata.
+
+### 2. [What kinds of activities could your specification become a part of that you are not designing for?](https://w3ctag.github.io/societal-impact-questionnaire/#unexpected-use)
+WoT does not directly address accessibility but IoT systems generally can be
+used to build custom user interfaces, including those that make systems easier
+to use (e.g. large buttons to replace difficult to use controls) or map sensory
+modalities (e.g. mapping visual indicators to audible ones or vice-versa).
+Unfortunately, the current WoT specification describes only network behavior,
+not physical behavior, making automated construction of such interfaces difficult.
+However, since WoT uses JSON-LD, it would be possible in theory to annotate WoT
+Thing Descriptions with information such as this.
+
+In general, additional metadata can be attached to WoT Thing Descriptions and
+this can be used for arbitrary purposes that can go beyond what is currently
+in the specification.  For example, a given device manufacturer could attach
+geolocation information to a WoT Thing Description.  To mitigate privacy
+hazards from both current and future metadata in WoT Thing Descriptions, the 
+WoT Discovery process provides mechanisms to control access to WoT Thing Descriptions.
+
+### 3. [What risks do you see in features of your specification being misused, or used differently from how you intended?](https://w3ctag.github.io/societal-impact-questionnaire/#misuse)
+Many possible misuses and risks are already discussed in the answers to the Security and
+Privacy questionnaire, such as distributed denial of service attacks and
+man-in-the-middle attacks.  See also the individual specifications for 
+descriptions of risks and mitigations.  
+
+In general, in addition to the "normal" risks of access to digital information,
+IoT devices can also provide real-time sensor data (a surveillance risk), and
+IoT actuators could potentially be a safety risk (e.g. an attacker starting a car in
+an enclosed space, filling the garage with carbon monoxide).  Most of these
+already exist without WoT, so what WoT brings is easier access to this functionality.
+It describes what already exists, it does not create it.  Mechanisms provided
+in WoT Discovery, however, can be used to limit access to WoT metadata to those
+who already have access to the associated IoT services.
+
+### 4. [Can users of the Web Platform choose not to use features of your specification?](https://w3ctag.github.io/societal-impact-questionnaire/#opt-out)
+An end user can choose not to install IoT devices.
```

Comment:
I think, the answer should like that this depends on the application running on the Web Platform. If the app expects WoT-enabled features than I would say this is kind of mandatory. However, if the app is more flexible to use also non-WoT features then this would be also possible.

Reply:
not sure what changes you wanted here but I updated the text to try and capture your point.  I also added text to capture the manufacturers' point of view.